src/main/webapp/education.html

```diff
@@ -4,33 +4,142 @@
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    
+        <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
         <script src="stats.js"></script>
+        <script src="protests.js"></script>
     </head>
 
     <body>
-        <h1>About Black Lives Matter</h1>
+        <div>
+        <!-- Navigation -->
+        <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+            <div class="container">
+                <a class="navbar-brand" href="#">[Website Name/Icon]</a>
+
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
 
-        <table class="columns">
-          <tr>
-            <td><div id="population_chart" style="border: 1px solid #ccc"></div></td>
-            <td><div id="stats_chart" style="border: 1px solid #ccc"></div></td>
-          </tr>
-        </table>
-        <section class="content"><p> Black Lives Matter Text Goes Here</p></section>
+                <div class="collapse navbar-collapse" id="navbarResponsive">
+                    <ul class="navbar-nav ml-auto">
+                        <li class="nav-item active">
+                            <a class="nav-link" href="./index.html">Home
+                                <span class="sr-only">(current)</span>
+                            </a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="./education.html">Education</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="./business.html">Businesses</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="./donations.html">Donations</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        </div>
+        <img class="image-banner" src="images/blmbanner.jpeg"/>
+        <div class="center">
+            <p>Black Lives Matter is an organized movement that advocates for civil protest
+            against acts of systemic anti-Black racism and state-sanctioned violence that disproportionately 
+            affects Black lives. The founders Alice Garza, Patrisse Khan-Cullors, and Opal Tometi started the 
+            movement to connect all those with a shared interest to fight injustices against the Black community 
```

Comment:
Would be nice to have wikipedia links for the founders, if it exists. No need to change for this PR.

Reply:
That's a good idea. Thanks for the suggestion.